integration_test.go

```diff
@@ -0,0 +1,935 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
Just curious: are these tests modified from the client lib?

Reply:
No, these are the tests that were already in this library, but they were spread out over a number of different files. Also, they were always executed, also when you executed the tests with the `-short` option. This PR bundles all the existing integration tests in this library into one file and makes sure they are not executed in `-short` mode (so that part is retrofitted from the client library).

Reply:
Sounds good.